ui/app/templates/components/oidc/provider-list.hbs

```diff
@@ -11,6 +11,10 @@
           <span class="has-text-weight-semibold is-underline">
             {{provider.name}}
           </span>
+          <div class="has-text-grey is-size-8">
+            Issuer:
+            {{provider.issuer}}
```

Comment:
Is there a change the issuer would be empty? If so do we need a placeholder for that situation?

Reply:
I don't think the issuer would ever be empty since if no issuer is set the default is an API with Vaul'ts
<img width="1054" alt="Screen Shot 2022-08-16 at 12 46 20 PM" src="https://user-images.githubusercontent.com/57650314/184934178-81ff1827-c6df-46cb-b7ca-662c1bd4b9f9.png">

Reply:
Yep - it's a value the backend will generate every time!